### PR TITLE
Vpda flag probe

### PR DESCRIPTION
**Pull request by shamoya**

### CODE REVIEW

File: drivers/net/mlx5/mlx5_vdpa.c [hunk 1]
```diff
@@ -280,6 +280,18 @@ mlx5_vdpa_pci_remove(struct rte_pci_device *pci_dev __rte_unused)
 }
 
 static const struct rte_pci_id mlx5_vdpa_pci_id_map[] = {
+	{
```
Comment (shahafsh) [inline drivers/net/mlx5/mlx5_vdpa.c:300]: how about having all the devices exists on the net PMD? not only VFs.

**Review by shahafsh — COMMENTED** [review]
Hi ido,

only small comment.

File: drivers/net/mlx5/mlx5_vdpa.c (hunk 1 above)
Comment (shamoya) [reply drivers/net/mlx5/mlx5_vdpa.c:300]: Thanks @shahafsh 
Do u mean add here all the devices (and not only VFs)?
the net PMD exits on all devices with vdpa=1 devarg ...

Comment (shahafsh) [reply drivers/net/mlx5/mlx5_vdpa.c:300]: yes, since it is for dev purpose, let's have it all.

Comment (shamoya) [reply drivers/net/mlx5/mlx5_vdpa.c:300]: ok

**Pull request merged**